src/measure_theory/probability_mass_function/basic.lean

```diff
@@ -66,9 +67,17 @@ section pure
   The value of `pure a` is `1` at `a` and `0` elsewhere. -/
 def pure (a : α) : pmf α := ⟨λ a', if a' = a then 1 else 0, has_sum_ite_eq _ _⟩
 
-@[simp] lemma pure_apply (a a' : α) : pure a a' = (if a' = a then 1 else 0) := rfl
+variables (a a' : α)
+
+@[simp]
+lemma pure_apply : pure a a' = (if a' = a then 1 else 0) :=
+rfl
```

Comment:
I would prefer to keep all these lemmas 1-liners. The more trivialities there fit on my screen, the better.